### PR TITLE
Add support for SSL settings to ElasticSearch output plugin

### LINKED ISSUE #3129: Add options on Elasticsearch output plugin to configure SSL settings.
## Feature Request
Add options on Elasticsearch output plugin to configure SSL. 
### Proposal:
Elasticsearch output plugin will have the similar options of influxdb output plugin for configuring ssl connections.

### Current behavior:
Elasticsearch output plugin fails when connecting to Elasticsearch protected by Searchguard (SSL enabled).

Error:
E! Failed to connect to output elasticsearch, retrying in 15s, error was 'no Elasticsearch node available'
### Desired behavior:
Options for 
```
# Optional SSL Config
# ssl_ca = "/etc/telegraf/ca.pem"
# ssl_cert = "/etc/telegraf/cert.pem"
# ssl_key = "/etc/telegraf/key.pem"
## Use SSL but skip chain & host verification
# insecure_skip_verify = false
```
### Use case: [Why is this important (helps with prioritizing requests)]
If the ouput.elasticsearch plugin doesn't have the option, then it cannot work with most production instances of elasticsearch secured by SSL.

### PR DESCRIPTION
**Pull request by adityacs**
Closes #3129 

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

**Comment by adityacs** [comment]
Will update unit tests.

File: plugins/outputs/elasticsearch/elasticsearch.go [hunk 1]
```diff
@@ -3,19 +3,20 @@ package elasticsearch
...
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
 )
 
 type Elasticsearch struct {
 	URLs                []string `toml:"urls"`
+	Scheme              string
```
Comment (danielnelson) [inline plugins/outputs/elasticsearch/elasticsearch.go]: Unused, I think this is cruft from development.

File: plugins/outputs/elasticsearch/elasticsearch.go [hunk 2]
```diff
@@ -76,7 +88,22 @@ func (a *Elasticsearch) Connect() error {
 
 	var clientOptions []elastic.ClientOptionFunc
 
+	tlsCfg, err := internal.GetTLSConfig(a.SSLCert, a.SSLKey, a.SSLCA, a.InsecureSkipVerify)
+	if err != nil {
+		return err
+	}
+	tr := &http.Transport{
+		ResponseHeaderTimeout: a.Timeout.Duration,
```
Comment (danielnelson) [inline plugins/outputs/elasticsearch/elasticsearch.go]: Let's remove the header timeout, I think we usually don't usually need this but it keeps getting copied around from other plugins...

**Comment by adityacs** [comment]
@danielnelson Have made the changes. Thanks for the review.

**Pull request merged**